### ISSUE TITLE
Disable -SS by default

### ISSUE THREAD

**Issue by jwillemsen** [objectcomputing/OpenDDS#2098]
As far as I can tell the tao_idl -SS option (suppress S.cpp) is only generated when the safety profile is enabled, as this file is for TAO server side, it should very likely be added as default option, saves code generation and footprint.

**Comment by mitza-oci** [comment]
I think a pull request for this would be uncontroversial.

**Comment by jwillemsen** [comment]
Yes, a todo, but as you can see from my other activity today we are working hard on various challenges related to OpenDDS

**Comment by jwillemsen** [comment]
At the moment InfoRepo is enabled very likely the -SS should not be used for all core DDS IDL, at the moment InfoRepo is not enabled it can be done as general flag.
